jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TravelerInputData.java

```diff
@@ -1,152 +1,162 @@
+        public String infoString;
+        public long packetID;
+        public String content;
+        public String[] items;
+        public short sspTimRights;
+        public short sspMsgTypes;
+        public short sspMsgContent;
+        public short sspLocationRights;
+        public int mutcd;
+        public int priority;
+        public String startTime;
+        public String startYear;
+        public int durationTime;
+        public String[] heading;
+        public int infoType;
+        public Region[] regions;
+        public RoadSign roadSign;
+        public String url;
+
+
+        public static class Region {
+            public String regionType;
+            public LaneNode[] laneNodes;
+            public int extent = -1;
+//            public GeoPoint refPoint;
+
+            @Override
+            public String toString() {
+                return "Region [regionType=" + regionType
+                        + ", laneNodes=" + Arrays.toString(laneNodes)
+                        + ", extent=" + extent
+//                        + ", refPoint=" + refPoint
+                        + "]";
+            }
+        }
+
+        public static class RoadSign {
+            public double latitude;
+            public double longitude;
+            public double elevation;
+            public String[] viewAngle;
+            public String mutcdCode;
+            public String msgCrc;
+
+            @Override
+            public String toString() {
+                return "Road Sign [latitude=" + latitude
+                        + ", longitude=" + longitude
+                        + ", elevation=" + elevation
+                        + ", viewAngle=" + viewAngle
+                        + ", mutcdCode=" + mutcdCode
+                        + ", msgCrc=" + msgCrc
+                        + "]";
+            }
+        }
+
+        @Override
+        public String toString() {
+            return "AnchorPoint [name=" + name + ", referenceLat="
+                    + latitude + ", referenceLon=" + longitude
+                    + ", referenceElevation=" + elevation
+                    + ", masterLaneWidth=" + masterLaneWidth
+                    + ", sspTimRights=" + sspTimRights
+                    + ", sspTypeRights=" + sspMsgTypes
+                    + ", sspContentRights=" + sspMsgContent
+                    + ", sspLocationRights=" + sspLocationRights
+                    + ", packetID=" + packetID
+                    + ", RoadSign=" + roadSign
+                    + ", content=" + content
+                    + ", items=" + Arrays.toString(items)
+                    + ", mutcd=" + mutcd
+                    + ", priority=" + priority
+                    + ", startTime=" + startTime
+                    + ", heading=" + Arrays.toString(heading)
+                    + ", infoType=" + infoType
+                    + ", regions="+Arrays.toString(regions)+"]";
+        }
+    }
+
+    public static class Deposit {
```

Comment:
Deposit class should probably be outside of this class in the same level as RSU and SNMP structures. We have one already defined for ODE-147 story that we could use at the top level of the RESTful request body.